### PR TITLE
add operator to the installer payload

### PR DESCRIPTION
**Pull request by sanchezl**
Adds the cluster-kube-storage-version-migrator-operator to the installer payload. The operator will be inert until its enabled in a followup PR.

### CODE REVIEW

File: bindata/kube-storage-version-migrator/deployment.yaml [hunk 1]
```diff
@@ -18,10 +18,9 @@ spec:
...
       - name: migrator
-        image: quay.io/sanchezl/storage-version-migration-migrator:v0.1
+        image: ${IMAGE}
         command:
           - /migrator
           - '--alsologtostderr'
-        args:
-          - '--v=5'
         terminationMessagePolicy: FallbackToLogsOnError
+        imagePullPolicy: Always
```
Comment (deads2k) [inline bindata/kube-storage-version-migrator/deployment.yaml]: If not present

File: manifests/0000_40_kube-storage-version-migrator-operator_07_deployment.yaml [hunk 2]
```diff
@@ -22,26 +22,42 @@ spec:
       containers:
       - name: kube-storage-version-migrator-operator
         terminationMessagePolicy: FallbackToLogsOnError
-        image: quay.io/sanchezl/storage-version-migration-migrator:v0.1
-        imagePullPolicy: IfNotPresent
+        image: quay.io/sanchezl/origin-cluster-kube-storage-version-migrator-operator:4.3
```
Comment (deads2k) [inline manifests/0000_40_kube-storage-version-migrator-operator_07_deployment.yaml]: Questionable

File: manifests/0000_40_kube-storage-version-migrator-operator_07_deployment.yaml [hunk 3]
```diff
@@ -22,26 +22,42 @@ spec:
...
+        imagePullPolicy: Always
```
Comment (deads2k) [inline manifests/0000_40_kube-storage-version-migrator-operator_07_deployment.yaml]: If not present

File: pkg/operator/targetcontroller/sync.go [hunk 4]
```diff
@@ -170,25 +175,25 @@ func (c *TargetController) manageKubeStorageVersionManagerDeployment(spec *opera
...
-				container.Image = c.operatorImagePullSpec
-			default:
-				if strings.Contains(container.Image, "$") {
-					return fmt.Errorf("invalid image reference %q", container.Image)
-				}
+func (c *TargetController) resolveImageReferences(containers []corev1.Container) ([]corev1.Container, error) {
+	var results []corev1.Container
+	for _, container := range containers {
+		switch container.Image {
+		case "${IMAGE}":
```
Comment (deads2k) [inline pkg/operator/targetcontroller/sync.go]: You don't do this, the cvo does it

**Comment by deads2k** [comment]
If it passes e2e aws, this is better than nothing for gaining traction

/lgtm

**Comment by sanchezl** [comment]
> If it passes e2e aws, this is better than nothing for gaining traction
Thanks. The image names and pull policy will be fixed in an update. I was still working out some issues with the image builds.

**Pull request merged**